main.py

```diff
@@ -113,6 +113,14 @@ def get(self):
 def get_redirect_route(regex_route, handler, defaults=None):
     """Returns a route that redirects /foo/ to /foo.
 
+    Args:
+        regex_route: str. Use for regex routing.
```

Comment:
Explain what this string is, or what it describes -- not just what it's used for.